k8s/bigchaindb/bigchaindb-ss.yaml

```diff
@@ -118,7 +118,7 @@ spec:
               key: bdb-rpc-port       
         resources:
           limits:
-            cpu: 200m
+            cpu: 1
```

Comment:
What does this mean (`cpu: 1`)?

Reply:
Also, why wasn't the corresponding `cpu:` line (line 285) changed for the bigchaindb container?

Reply:
does one core means more compute resource than 200m?

Reply:
:+1: